how-to-go-home-early-github.md

```diff
@@ -0,0 +1,75 @@
+
+To set this up, go to your GitHub repo with actions enabled, click on "New workflow" and once in the marketplace scroll down to "Automate every step in your process" where you'll find the "labeler" action:
+
+![Alt screenshot of GitHub action in marketplace](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/labeler-marketplace.png)
+
+### Automatically request reviews with "Code owners" 
+
+We can take the automatic labeling of your PR one step further. There is a feature in GitHub called "Code owners" which let's you assign co-workers to certain parts of your application(s) GitHub will use codeowners to automatically asign reviews.
+
+![Alt screenshot of reviewers being assigned](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/codeowners.png)
+
+Setting up code owners is easy, check [this guide](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) to get started with code owners.
+
+### Let a panda spam your co-workers into submission for reviews.
+
+One thing that helped our team a lot was the addition of [Pull panda](https://pullreminders.com/). It is a Slack integration that automatically sends a message to your co-workers when someone requests their review. 
+
+When you request a review, the reviewer gets a message like:
+
+![Alt screenshot of pull panda request message](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/panda-assigned.png)
+
+If someone approves your pull request, you'll be notified as well:
+
+![Alt screenshot of pull panda request message](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/panda-approved.png)
+
+Even if someone comments on your pull request, panda will show you the comments right away in Slack:
+
+![Alt screenshot of pull panda request message](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/panda-comments.png)
+
+### Automatically merge the pull request
+
+Our pull request got labeled automatically, the correct reviewers got assigned automatically and they got a Slack message reminding them that there is work to be done. Now there is one last thing ~~we~~ the robots should do, merge the pull request into the master branch! If your team uses any form of automated checks on pull requests, you have probably already wasted some time blindly staring at your screen until the merge button turns green.
+
+We can even automate the merging of our pull request with another GitHub action called [automerge](https://github.com/pascalgn/automerge-action). This action automatically merges your pull request based on your team's preferences.
+
+![Alt screenshot of GitHub merging the pull request](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/automerge.png)
+
+We configured it so that it merges a pull request once:
```

Comment:
This sentence is a bit dutch to my ears. I would replace it with something like:
```suggestion
We set up automerge to merge a pull request once:
```